src/compass/utils/geo_metadata.py

```diff
@@ -265,3 +265,181 @@ def to_file(self, dst, fmt:str):
+                              {'description': 'Sentinel platform ID (S1, S2)'})
+
+        center_lon_lat = np.array([val[0] for val in self.center.coords.xy])
+        add_dataset_and_attrs(metadata_group, 'center',
+                              center_lon_lat,
+                              {'description': 'coordinates of center of burst',
+                               'units': 'degrees'})
+
+        # list of lon, lat coordinate tuples (in degrees) representing burst border
+        border_group = metadata_group.require_group('border')
+        border_x, border_y = self.border.exterior.coords.xy
+        add_dataset_and_attrs(border_group, 'x', border_x,
+                              {'description': 'list boundary x-coordinates',
+                               'units': 'degrees'})
+        add_dataset_and_attrs(border_group, 'y', border_y,
+                              {'description': 'list boundary y-coordinates',
+                               'units': 'degrees'})
+
+        orbit_group = metadata_group.require_group('orbit')
+        self.orbit.save_to_h5(orbit_group)
+
+        add_dataset_and_attrs(metadata_group, 'orbit_direction',
+                              np.string_(self.orbit_direction),
+                              {'description':'direction of orbit (ascending, descending)'})
+
+        # VRT params
+        add_dataset_and_attrs(metadata_group, 'tiff_path',
+                              np.string_(self.tiff_path),
+                              {'description': 'path to TIFF containing the burst'})
+        add_dataset_and_attrs(metadata_group, 'i_burst', self.i_burst,
+                              {'description': 'Sentinel platform ID (S1, S2)'})
+        # window parameters
+        add_dataset_and_attrs(metadata_group, 'range_window_type',
+                              np.string_(self.range_window_type),
+                              {'description': 'name of the weighting window type used during processing'})
+        add_dataset_and_attrs(metadata_group, 'range_window_coefficient',
+                              self.range_window_coefficient,
+                              {'description': 'value of the weighting window coefficient used during processing'})
+
+        geogrid_group = metadata_group.require_group('geogrid')
+        add_dataset_and_attrs(geogrid_group, 'start_x', self.geogrid.start_x,
+                              {'description': 'starting y coordinate of geogrid output',
+                               'units': 'consistent with EPSG'})
+        add_dataset_and_attrs(geogrid_group, 'start_y', self.geogrid.start_y,
+                              {'description': 'starting y coordinate of geogrid output',
+                               'units': 'consistent with EPSG'})
+        add_dataset_and_attrs(geogrid_group, 'spacing_x',
+                              self.geogrid.spacing_x,
+                              {'description': 'x spacing coordinate of geogrid output',
```

Comment:
```suggestion
                              {'description': 'Spacing along X-direction of output geographical grid',
```
Same for Y-direction. Note that if we use the burst data base to process these spacings will be in meters, so please correct the units